nefertari/events.py

```diff
@@ -0,0 +1,198 @@
+from contextlib import contextmanager
+
+from zope.interface import (
+    Attribute,
+    Interface,
+)
+from zope.interface import implementer
+
+
+class IRequestEvent(Interface):
```

Comment:
Got it. I guess I added this interface at the very beginning because I thought they may be various implementations.